lib/_http_outgoing.js

```diff
@@ -1005,6 +1019,120 @@ function connectionCorkNT(conn) {
   conn.uncork();
 }
 
+OutgoingMessage.prototype.sendFile = function sendFile(resource, optsOrCallback, callbackOrNil) {
```

Comment:
We could also call this `send` in which case `resource` could also be a http url?